Vignettes/baseline/baseline.Rnw

```diff
@@ -143,10 +143,10 @@ The baselines and respective supporting points for each iteration of
 are shown in figure~\ref{fig:iter}.
 
 <<trace, echo = FALSE, results = hide>>=
-trace (spc.fit.poly.below, quote ({
+trace (spc.fit.poly.below, exit = quote ({
   plot (fit.to[,, use.old], col = cl, add = TRUE, lines.args = list (pch = 20, type = "p"));
   lines (fit.to@wavelength, bl, col = cl);
-}), at = 12, print = FALSE)
+}), print = FALSE)
```

Comment:
There's  a much better version in the fix that I forgot to push:  I added a `debuglevel` parameter that triggers plotting.